src/xSTIR/cSTIR/include/sirf/STIR/stir_data_containers.h

```diff
@@ -1362,6 +1362,12 @@ namespace sirf {
 			const Coord3DI& new_sizes = { -1,-1,-1 },
 			const stir::ZoomOptions zoom_options = stir::ZoomOptions::preserve_sum);
 
+		void zoom_image(const STIRImageData& original_image,
+			const stir::ZoomOptions zoom_options = stir::ZoomOptions::preserve_sum);
+
+		void zoom_image(const STIRImageData& original_image, 
+		    const char* const zoom_options_str = "preserve_sum");
```

Comment:
naming of parameter would have to be `template_image`. Needs some doxygen as well. Finally, I'd prefer these to be called exactly the same as the Python version. e.g. `zoom_image_as_template`